src/client/runner/runner.js

```diff
@@ -176,8 +172,18 @@ Runner.prototype._onFatalError = function (err) {
 
     RunnerBase.prototype._onFatalError.call(this, err);
 
-    this._onTestError(err)
-        .then(Runner.checkStatus);
+    // NOTE: we should not stop the test run if an error occured during page unloading because we
```

Comment:
Replace with:
"// NOTE: we should not stop the test running if an error occurs during page unload. As a result, we
// would destroy the session and wouldn't be able to get the next page in the browser.
// We have to set the deferred error to the task to have the test fail after page reload."